atom/browser/ui/tray_icon_cocoa.mm

```diff
@@ -90,7 +90,7 @@ - (void)drawRect:(NSRect)dirtyRect {
   CGFloat thickness = [[statusItem_ statusBar] thickness];
 
   // Draw the system bar background.
-  [statusItem_ drawStatusBarBackgroundInRect:self.bounds withHighlight:highlight];
+  [statusItem_ drawStatusBarBackgroundInRect:self.bounds withHighlight:shouldHighlight];
```

Comment:
According to the larger diff, I _think_ `highlight` and `shouldHiglight` have the same value here:

```objc
  BOOL highlight = [self shouldHighlight];
  BOOL highlightContent = highlight | [self isDarkMode];
  CGFloat thickness = [[statusItem_ statusBar] thickness];

  // Draw the system bar background.
  [statusItem_ drawStatusBarBackgroundInRect:self.bounds withHighlight:shouldHighlight];
```

So, while this patch [LGTM in master](https://github.com/electron/electron/pull/12395/files), does it actually change any behavior here?